server/src/test/scala/za/co/absa/atum/server/api/service/PartitioningServiceIntegrationTests.scala

```diff
@@ -56,6 +57,11 @@ class PartitioningServiceIntegrationTests extends ZIOSpecDefault with TestData {
   when(partitioningRepositoryMock.getPartitioningAdditionalData(partitioningDTO2))
     .thenReturn(ZIO.fail(DatabaseError("boom!")))
 
+  when(partitioningRepositoryMock.getPartitioningCheckpoints(checkpointQueryDTO1))
```

Comment:
Rename the class as these are unit tests and therefore should be executed as such. Also change it to object and remove the runwith annotation.